Jenkinsfile

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env groovy
+
+/* Only keep the 10 most recent builds. */
+properties([[$class: 'BuildDiscarderProperty',
+                strategy: [$class: 'LogRotator', numToKeepStr: '10']]])
+
+
+node('java') {
+    timestamps {
+        stage('Checkout') {
+            checkout scm
+        }
+
+        stage('Build') {
+            withEnv([
```

Comment:
isn't `withMaven` supposed to set all this up for you (with the advantage of knowing about the settings) (not that I like the `withMaven` step... but still)